eosq/src/components/footer/footer.component.tsx

```diff
@@ -77,19 +77,18 @@ const BaseFooter = () => (
               q
             </LogoSecond>
           </LogoLink>
-          <Cell pt={[3]}>
-            <Text color={theme.colors.primary}>{t("footer.subTitle")}</Text>
-          </Cell>
-          <ExternalTextLink to="https://eoscanada.com">
-            <StyledLogoImg src="/images/logo-eos-canada.svg" />
-          </ExternalTextLink>
+          {/* <Cell pt={[3]}>
```

Comment:
@phila82r Don't leave dead code around, please open another PR to remove all these.

Reply:
I wasn't sure if these were gonna be used again at some point (this is related to "subTitle" not being used), so I kept them there. So what you're saying is that, whatever the case, if some of the code is not used it should be removed entirely?

Reply:
Yes exact, Git is going to be there if we need to go back in time.